pkgs/data/icons/google-cursor/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenvNoCC
+, fetchzip
+, lib
+}:
+
+let
+  colors = [
+    {
+      name = "Black";
+      hash = "sha256-pb2U9j1m8uJaILxUxKqp8q9FGuwzZsQvhPP3bfGZL5I=";
+    }
+    {
+      name = "Blue";
+      hash = "sha256-PmJeGShQLIC7ceRwQvSbphqz19fKptksZeHKi9QSL5Y=";
+    }
+    {
+      name = "Red";
+      hash = "sha256-/X81jLoWaw4UMoDRf1f6oaKKRWexQc4PAACy3doV4Kc=";
+    }
+    {
+      name = "White";
+      hash = "sha256-eT/Zy6O6TBD6G8q/dg+9rNYDHutLLxEY1lvLDP90b+g=";
+    }
+  ];
+in
+stdenvNoCC.mkDerivation rec {
```

Comment:
Just noticed I didn't catch this earlier, please remove `rec` and use `finalAttrs` where needed (which allows proper overriding).

See https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes

Reply:
Oh sure, I'll do that tomorrow.

Reply:
Fixed